Documentation/Testing/CoreTesting.rst

```diff
@@ -9,20 +9,17 @@ Core testing
 Introduction
 ============
 
-This chapter is mostly about executing TYPO3 core tests locally and getting an idea of the overall
-picture and how things play with each other. A full core git checkout comes with everything needed
-to run tests in TYPO3 since core version 9. We don't look back to older versions in this chapter
+This chapter is about executing TYPO3 core tests locally and is intended to give you a better understanding of testing within TYPO3's core. A full core git checkout comes with everything needed
+to run tests in TYPO3 as of version 9. We don't use older versions in this chapter
 since core development is most likely bound to the core master branch - back porting patches to older
-branches are usually handled by core maintainers and often don't affect other core contributors much.
+branches are usually handled by core maintainers and often don't affect other core contributors.
 
-As a disclaimer, the main script :file:`Build/Scripts/runTests.sh` is relatively young. It works best
-if executed on a linux based host, it works using macOS (but much slower), and it works using
-Windows (and it's actually quicker on Windows than with mac). Not everything in this area is settled yet,
-though.
+Note, the main script :file:`Build/Scripts/runTests.sh` is relatively new. It works best
+when executed on a Linux based host but can be run under macOS and Windows with some performance drawbacks.
```

Comment:
Correct would be: ... macOS (with some performance drawbacks) and Windows. 

The performance problems only apply to macOS

Reply:
Fair comment, I've updated the paragraph.